### ISSUE TITLE
Closing notepad++ by middle clicking the window preview causes it to hang

### ISSUE THREAD

**Issue by jamietwells** [notepad-plus-plus/notepad-plus-plus#7937]
### Description of the Issue
Whenever I close all my open windows at the end of the day I go along the taskbar and middle-click the preview windows for all the open applications. When I get to Notepad++ it often hangs, requiring the Task Manager.

### Steps to Reproduce the Issue
1. Open Notepad++
2. Open an existing file
3. Make an unsaved change
4. Minimize Notepad++ using the buttons at the top
5. Middle-click the Notepad++ preview in the taskbar

### Expected Behavior
You should be prompted to save or discard changes

### Actual Behavior
Notepad++ becomes unresponsive and must be ended using Task Manager

### Debug Information
Notepad++ v7.8.2   (32-bit)
Build time : Dec  4 2019 - 01:36:59
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS Name : Windows 10 Enterprise (64-bit) 
OS Version : 1809
OS Build : 17763.864
Plugins : JSMinNPP.dll mimeTools.dll NppConverter.dll NppExport.dll NppTextFX.dll 

![Image showing the area I middle click to close windows](https://user-images.githubusercontent.com/28688077/74262253-706acc80-4cf4-11ea-9cf8-18d307cf216d.png)

**Comment by Ekopalypse** [comment]
This also happens when using the left button and the close icon from the preview.
Reason seems to be that the save dialog moves to a non-visible area.

![image](https://user-images.githubusercontent.com/47723516/74264053-50410a80-4d00-11ea-9c45-3738e5181b2a.png)

@sasumner , @donho maybe another candidate for the make-dialog-always-visible code? :)

**Comment by sasumner** [comment]
Duplicate of #7871 (at least I think so)

**Comment by jamietwells** [comment]
Yes, looks like it is. Thank you. Is there a fix on the way? I'm not sure my c++ skills are up to the task of making a PR!

**Comment by sasumner** [comment]
@Ekopalypse said:

>maybe another candidate for the make-dialog-always-visible code? :)

This is a reference to #905 where code to remember the find-window's position across Notepad++ runs also checked for, and corrected for, a find-window that happened to be offscreen and not viewable/grabbable by the user.

**Comment by sasumner** [comment]
@jamietwells said:

> Is there a fix on the way?

Well, with an open issue (or two) there's always some hope! :-)
